package.json

```diff
@@ -211,38 +211,42 @@
     }
   },
   "scripts": {
-    "vscode:prepublish": "npm run compile && npm run tslint",
-    "compile": "tsc -p ./",
-    "watch": "tsc -watch -p ./",
+    "vscode:prepublish": "webpack --mode production",
+    "compile": "webpack --mode none",
+    "watch": "webpack --mode none --watch --info-verbosity verbose",
     "postinstall": "node ./node_modules/vscode/bin/install",
-    "test": "npm run compile && node ./node_modules/vscode/bin/test",
+    "test": "tsc -p . && node ./node_modules/vscode/bin/test",
```

Comment:
The compile command is now `webpack --mode none`. Is this first step for the tests correct?

Reply:
`tsc -p .` will still compile everything (including tests) to `out/`, which I'm guessing is where `./node_modules/vscode/bin/test` picks up the test `.js` files. I tried `webpack --mode none` instead of `tsc -p .` but the VS Code test command could not find the tests.